assets/js/app.js

```diff
@@ -21,11 +21,17 @@ import 'phoenix_html'
 import { Socket } from 'phoenix'
 import { LiveSocket } from 'phoenix_live_view'
 import topbar from '../vendor/topbar'
+import CopyHook from './hooks/copy_hook.js'
+
+const Hooks = {
+  ...{ CopyHook },
+}
```

Comment:
Unnecessarily complicated code
```suggestion
const Hooks = {
  CopyHook,
}
```